doc/modules/cross_validation.rst

```diff
@@ -467,44 +376,189 @@ Here is a usage example::
 validation that allows a finer control on the number of iterations and
 the proportion of samples on each side of the train / test split.
 
+Cross-validation iterators with stratification based on class labels.
+=====================================================================
+
+For classification tasks, in some datasets the number of samples per class
+varies based on the class label. They are referred to commonly as datasets
+with imbalanced classes. In such cases, stratified sampling may be done to
+split the data in cross-validation.
```

Comment:
Cross-validation always splits the data with or without stratified sampling.

Instead, let's simplify the formulation of this paragraph:

Some classification problems can exhibit large imbalance in the distribution of classes: for instance  there could be several times more negative samples than positive samples. In such cases it is recommended to use stratified sampling as implemented in :class:`StratifiedKFold` and :class:`StratifiedShuffleSplit` to ensure that relative class frequencies is approximately preserved in each train and validation fold.